packages/core/src/auth/auth.ts

```diff
@@ -121,6 +121,13 @@ export interface ConnectionStateChangeEvent {
 
 export type AuthType = Auth
 
+type DeclaredConnection = {
```

Comment:
Is it possible to use the `SsoProfile` interface instead? Or make a new interface which extends `SsoProfile`. This type and SsoProfile are semantically related. If we do not even need the `id` then maybe we can use `SsoProfile` itself.

Reply:
done